frontend/src/lib-customizations/defaults/citizen/i18n/sv.tsx

```diff
@@ -368,6 +368,9 @@ const sv: Translations = {
       selectedChildren: 'Utvalda barn',
       selectChildrenInfo:
         'Anmäl endast heldagsfrånvaro. Kortare frånvaro kan anmälas genom reservationer.',
+      lockedAbsencesWarningTitle: 'Frånvaro i flera dagar',
```

Comment:
```suggestion
      lockedAbsencesWarningTitle: 'Frånvaro för flera dagar',
```
Not sure about `för` vs `i` vs `på` here, but I think that at least this text and the longer text should use the same word 🤷 

Perhaps these translations should be checked anyway with someone who actually knows the language.

Reply:
the text came from svebi 😅